telegraf/nightly/entrypoint.sh

```diff
@@ -11,5 +11,5 @@ else
     # Allow telegraf to send ICMP packets and bind to privliged ports
     setcap cap_net_raw,cap_net_bind_service+ep /usr/bin/telegraf || echo "Failed to set additional capabilities on /usr/bin/telegraf"
 
-    exec setpriv --reuid telegraf --init-groups "$@"
+    exec setpriv --reuid telegraf --regid telegraf --reset-env --clear-groups "$@"
```

Comment:
As discussed elsewhere (and pointed out by you), `--reset-env` is too much and would cause regressions. Perhaps change this to:
```
export HOME=$(getent passwd telegraf | cut -d : -f 6)
exec setpriv --reuid telegraf --regid telegraf --groups telegraf "$@"
```